pilz_trajectory_generation/test/unittest_trajectory_generator_circ.cpp

```diff
@@ -302,50 +288,50 @@ TEST_P(TrajectoryGeneratorCIRCTest, toFast)
-
-  ROS_ERROR_STREAM("Loading point");
-
-  ASSERT_TRUE(tdp_->getCirc("CIRCCmdAllPointsTheSame", circ_cmd)) << "failed to get circ command from test data";
-  // construct request
-  moveit_msgs::MotionPlanRequest req = req_director_.getCIRCJointReq(robot_model_, circ_cmd);
+  // Define auxiliary point and goal to be the same as the start
+  auto circ {tdp_->getCircCartCenterCart("circ1_center_2")};
+  circ.getAuxiliaryConfiguration().getConfiguration().setPose(circ.getStartConfiguration().getPose());
+  circ.getAuxiliaryConfiguration().getConfiguration().getPose().position.x += 1e-8;
+  circ.getAuxiliaryConfiguration().getConfiguration().getPose().position.y += 1e-8;
+  circ.getAuxiliaryConfiguration().getConfiguration().getPose().position.z += 1e-8;
+  circ.getGoalConfiguration().setPose(circ.getStartConfiguration().getPose());
+  circ.getGoalConfiguration().getPose().position.x -= 1e-8;
+  circ.getGoalConfiguration().getPose().position.y -= 1e-8;
+  circ.getGoalConfiguration().getPose().position.z -= 1e-8;
 
   planning_interface::MotionPlanResponse res;
-  EXPECT_FALSE(circ_->generate(req,res));
+  EXPECT_FALSE(circ_->generate(circ.toRequest(),res));
   EXPECT_EQ(res.error_code_.val, moveit_msgs::MoveItErrorCodes::INVALID_MOTION_PLAN);
 }
 
-
 /**
  * @brief test invalid motion plan request with no aux point defined
  */
 TEST_P(TrajectoryGeneratorCIRCTest, emptyAux)
 {
-  // get the test data from xml
-  pilz_industrial_motion_testutils::STestMotionCommand circ_cmd;
-  circ_cmd.aux_pos_type = pilz_industrial_motion_testutils::ECircAuxPosType::eCENTER;
-  ASSERT_TRUE(tdp_->getCirc("ValidCIRCCmd1", circ_cmd)) << "failed to get circ command from test data";
-  moveit_msgs::MotionPlanRequest req = req_director_.getCIRCJointReq(robot_model_, circ_cmd);
+  // Define auxiliary point and goal to be the same as the start
+  auto circ {tdp_->getCircCartCenterCart("circ1_center_2")};
 
   // empty path constraint
+  planning_interface::MotionPlanRequest req = circ.toRequest();
+
   req.path_constraints.position_constraints.clear();
+
   planning_interface::MotionPlanResponse res;
   EXPECT_FALSE(circ_->generate(req,res));
-  EXPECT_EQ(res.error_code_.val, moveit_msgs::MoveItErrorCodes::INVALID_MOTION_PLAN);
+  EXPECT_EQ(res.error_code_.val, moveit_msgs::MoveItErrorCodes::INVALID_MOTION_PLAN) << req;
 }
 
 /**
- * @brief test invalid motion plan request with no aux point defined
+ * @brief test invalid motion plan request with no aux name defined
  */
 TEST_P(TrajectoryGeneratorCIRCTest, invalidAuxName)
 {
-  // get the test data from xml
-  pilz_industrial_motion_testutils::STestMotionCommand circ_cmd;
-  circ_cmd.aux_pos_type = pilz_industrial_motion_testutils::ECircAuxPosType::eCENTER;
-  ASSERT_TRUE(tdp_->getCirc("ValidCIRCCmd1", circ_cmd)) << "failed to get circ command from test data";
-  moveit_msgs::MotionPlanRequest req = req_director_.getCIRCJointReq(robot_model_, circ_cmd);
+  // Define auxiliary point and goal to be the same as the start
```

Comment:
Is this comment correct? I thought we do not save invalid commands. If the comment is incorrect please delete it here and in all other tests.